app/bin/service/search.dart

```diff
@@ -32,7 +32,7 @@ Future main() async {
   });
   for (int i = 0; i < envConfig.isolateCount; i++) {
     await Isolate.spawn(
-      _main,
```

Comment:
I mean, like an inline lambda `(msg) => _main(msg)`

does that not work?

Reply:
That gives you an `Isolate.spawn expects to be passed a static or top-level function` exception.

Reply:
Ugh. K